airflow/models/baseoperator.py

```diff
@@ -178,6 +178,9 @@ class derived from this one results in the creation of a task object,
     :param pool: the slot pool this task should run in, slot pools are a
         way to limit concurrency for certain tasks
     :type pool: str
+    :param pool_capacity: the number of pool slots this task should use (>= 1)
```

Comment:
Capacity is the overall size of the pool, and in the logs etc we talk about slots ("Not scheduling since there are %s open slots in pool %s") so do you think this would be better named as `pool_slots`

WDYT @tooptoop4 ?

Reply:
yes pool_slots sounds more reasonable. renamed pool_capacity to pool_slots